dagger-core/src/main/java/io/odpf/dagger/core/sink/kafka/KafkaSerializerBuilder.java

```diff
@@ -0,0 +1,7 @@
+package io.odpf.dagger.core.sink.kafka;
+
+import org.apache.flink.connector.kafka.sink.KafkaRecordSerializationSchema;
+
+public interface KafkaSerializerBuilder {
```

Comment:
how about moving these to common as well?

Reply:
same as above.